test/integration/package_int_test.go

```diff
@@ -783,6 +784,56 @@ func (suite *PackageIntegrationTestSuite) TestChangeSummary() {
 	cp.ExpectExitCode(0)
 }
 
+func (suite *PackageIntegrationTestSuite) TestBuildInProgress() {
+	suite.OnlyRunForTags(tagsuite.Package)
+	ts := e2e.New(suite.T(), false)
+	defer ts.Close()
+
+	ts.LoginAsPersistentUser()
+
+	// Publish a new ingredient revision, which, when coupled with `state install --ts now`, will
+	// force a build.
+	// The ingredient is a tarball comprising:
+	//   1. An empty, executable "configure" script (emulating autotools).
+	//   2. A simple Makefile with "all", "check", and "install" rules.
+	//   3. A simple "main.c" file, whose compiled executable prints "Hello world!".
+	cp := ts.Spawn("publish", "--non-interactive",
+		"--namespace", "private/"+e2e.PersistentUsername,
+		"--name", "hello-world",
+		"--version", "1.0.0",
+		"--depend", "builder/autotools-builder@>=0", // for ./configure, make, make install
+		"--depend", "internal/mingw-build-selector@>=0", // for Windows to use mingw's GCC
+		filepath.Join(osutil.GetTestDataDir(), "hello-world-1.0.0.tar.gz"),
+		"--edit") // publish a new revision each time, forcing a build
+	cp.Expect("Successfully published")
+	cp.ExpectExitCode(0)
+
+	cp = ts.Spawn("config", "set", constants.AsyncRuntimeConfig, "true")
+	cp.Expect("Successfully set")
+	cp.ExpectExitCode(0)
+
+	cp = ts.Spawn("checkout", "ActiveState-CLI/Perl-5.36", ".")
+	cp.Expect("Checked out")
+	cp.ExpectExitCode(0)
+
+	cp = ts.SpawnWithOpts(
+		e2e.OptArgs("install", "private/"+e2e.PersistentUsername+"/hello-world", "--ts", "now"),
```

Comment:
The JIRA ticket says to not use `--ts now`, but I think that is based on outdated behavior. I need `--ts now` to install the newest version just published in the lines above.